dataprofiler/data_readers/data_utils.py

```diff
@@ -4,6 +4,8 @@
 from io import open, StringIO, BytesIO, TextIOWrapper
 from collections import OrderedDict
 import dateutil
+import requests
+from urllib.parse import urlparse
```

Comment:
instead of importing specifically urlparse, we can just import urllib so we know where it's coming form below.

```python
import urllib

.
.
urllib.parse.urlparse(...)
```